packages/liveblocks-node/src/client.ts

```diff
@@ -520,13 +561,16 @@ export class Liveblocks {
    * Returns a room with the given id.
    * @param roomId The id of the room to return.
    * @returns The room with the given id.
+   * @param options.signal (optional) An abort signal to cancel the request.
    */
-  public async getRoom(roomId: string): Promise<RoomData> {
-    const res = await this.#get(url`/v2/rooms/${roomId}`);
+  public async getRoom(
+    roomId: string,
+    options?: RequestOptions
+  ): Promise<RoomData> {
```

Comment:
Do you think `getRoom()` (and perhaps, `getThread`, `getComment`) should return a `null` object if the room was not found? Not sure how big of a breaking change this would be though.

Reply:
I think that would be nice honestly. Dealing with custom errors from libraries is never a great DX, especially not when they may be "expected" errors like 404s. I'd prefer `null`, but ideally we'd make that a breaking change and write an upgrade guide in another release. I don't want to introduce a breaking change now.